Project 10/index.js

```diff
@@ -0,0 +1,128 @@
+const input = document.getElementById('input');
+let result;
+let operation = '';
+let deleteIfEnter = false;
+let clearMemory = false;
+
+const operations = {
+	'' : (value) => sumResult(value),
+	'+': (value) => sumResult(value),
+	'-': (value) => sumResult(-value),
+	'*': (value) => multiplyResult(value),
+	'/': (value) => divideResult(value),
+	'=': (value) => showResult(value)
+}
+
+function onload() {
+	document.addEventListener("keydown", keyDownInput, false);
+}
+
+function keyDownInput(e) {
+  let keyCode = e.keyCode;
+  let shift = e.shiftKey;
+  if (shift == false && keyCode >= 48 && keyCode <= 57) {
+    addNumberUI(keyCode - 48);
+    return;
+  }
+  if (keyCode == 191) {
+  	operate('/');
+  	return;
+  }
+  if (shift == true && keyCode == 58) {
+  	operate('*');
+  	return;
+  }
+  if (keyCode == 189) {
+  	operate('-');
+  	return;
+  }
+  if (shift == true && keyCode == 187) {
+  	operate('+');
+  	return;
+  }
+  if (keyCode == 187 || keyCode == 13) {
+  	operate('=');
+  	return;
+  }
+  if (keyCode == 8) {
+  	backspace();
+  	return;
+  }
+  if (keyCode == 67) {
+  	clearInput();
+  	return;
+  }
+}
+
+function clearInput() {
+	input.value = '';
+	if (!clearMemory) {
+		clearMemory = true;
+	} else {
+		clrMemory();
+	}
+}
+
+function clrMemory() {
```

Comment:
```suggestion
function clearMemory() {
```
хорошим тоном считается именовать сущности без сокращений, учитывая что в функции выше clear написано полностью